cmd/gb-vendor/vendor/repo.go

```diff
@@ -163,7 +163,8 @@ func probeGitUrl(schemes []string, host, path string, insecure bool) (string, er
 		switch scheme {
 		case "https":
 			url := scheme + "://" + host + "/" + path
-			if _, err := run("git", "ls-remote", "--exit-code", url, "HEAD"); err == nil {
```

Comment:
Let's go with that as it is a smaller change. We can go with your original
proposal if it proves to be insufficient.

On Sun, Jul 5, 2015 at 5:12 PM, Asbjørn Bjørnstad <notifications@github.com>
wrote:

> In cmd/gb-vendor/vendor/repo.go
> <https://github.com/constabulary/gb/pull/258#discussion_r33892555>:
>
> > @@ -163,7 +163,8 @@ func probeGitUrl(schemes []string, host, path string, insecure bool) (string, er
> >  		switch scheme {
> >  		case "https":
> >  			url := scheme + "://" + host + "/" + path
> > -			if _, err := run("git", "ls-remote", "--exit-code", url, "HEAD"); err == nil {
>
> Yes, that's probably fine. It should return error on non-existent
> repositories which is what's important, I guess?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/constabulary/gb/pull/258/files#r33892555>.
>